src/data-browser/charts/MapContainer.jsx

```diff
@@ -0,0 +1,354 @@
+  {value: 'race', label: 'Race'},
+  {value: 'ethnicity', label: 'Ethnicity'}
+]
+
+const valsForVar = {
+  loanType: optionsFromVariables('loan_types'),
+  loanPurpose: optionsFromVariables('loan_purposes'),
+  ethnicity: optionsFromVariables('ethnicities', 1),
+  race: optionsFromVariables('races', 1)
+}
+
+function optionsFromVariables(key, nameAsValue){
+  return VARIABLES[key].options.map( v => {
+    return {value: nameAsValue ? v.name : v.id, label: v.name}
+  })
+}
+
+function getValuesForVariable(variable) {
+  if(!variable) return []
+  return valsForVar[variable.value] || []
+}
+
+function getVariable(val){
+  for(let i=0; i<variables.length; i++){
+    if(val === variables[i].value) return variables[i]
+  }
+}
+
+function getValue(variable, val){
+  const vals = getValuesForVariable(variable)
+  for(let i=0; i<vals.length; i++){
+    if(val === vals[i].value) return vals[i]
+  }
+}
+
+function generateColor(data, variable, value, total) {
+  const count = data[variable][value]
+  const len = colors.length
+  const BIAS = 30
+  let index = Math.min(len-1, Math.floor(count/total*len*BIAS))
+  if(!index) index = 0
+  return colors[index]
+}
+
+function makeStops(data, variable, value){
+  const stops = [['0', 'rgba(0,0,0,0.05)']]
+  if(!data || !variable || !value) return stops
+  Object.keys(data).forEach(county => {
+    const currData = data[county]
+    const total = COUNTS[county] || 20000
+    stops.push([county, generateColor(currData, variable.value, value.value, total)])
+  })
+  return stops
+}
+
+function buildPopupHTML(data, features){
+  const fips = features[0].properties['GEOID']
+  return '<h4>' + fips + ' - ' + COUNTIES[fips] + '</h4>'
+}
+
+function getDefaultsFromSearch(props) {
+  const { search } = props.location
+  const qsParts = search.slice(1).split('&')
+  const defaults = {
+    variable: null,
+    value: null,
+    fips: null
+  }
+  qsParts.forEach(part => {
+    if(!part) return
+    let [key, val] = part.split('=')
+    if(key === 'variable') val = getVariable(val)
+    else if(key === 'value') val = getValue(defaults.variable, val)
+    defaults[key] = val || null
+  })
+  return defaults
+}
+
+function scrollToTable(node){
+  if(!node) return
+  node.scrollIntoView({behavior: 'smooth', block: 'end'})
+}
+
+const popup = new mapbox.Popup({
+  closeButton: false,
+  closeOnClick: false,
+  maxWidth: '750px'
+})
+
+const MapContainer = props => {
+  const mapContainer = useRef(null)
+  const tableRef = useRef(null)
+
+  const defaults = getDefaultsFromSearch(props)
+
+  const [map, setMap] = useState(null)
+  const [data, setData] = useState(null)
+  const [selectedVariable, setVariable] = useState(defaults.variable)
+  const [selectedValue, setValue] = useState(defaults.value)
+  const [fips, setFips] = useState(defaults.fips)
+
+  const fetchCSV = () => {
+    const csv = `/v2/data-browser-api/view/csv?years=2018&counties=${fips}&${selectedVariable.value}=${selectedValue.value}`
```

Comment:
When proxying against `dev-default` I am unable to download the CSV data.

When pointing to a locally running `data-browser-api` the downloaded data looks good.

Reply:
This looks like a problem with `dev-default`, you'll notice that csv downloads are failing in the  dataset filtering page as well